Config/Sample.cfg

```diff
@@ -104,6 +104,7 @@ SwitchThreadsToRtPriority       : 1             # 0: OFF, 1: Switch threads to r
 FPSInVPS                        : 0             # 0: OFF, 1: Enable VPS timing info 
 MaxCLL                          : 0             # SEI message (0=OFF,  1=ON ) 
 MaxFALL                         : 0             # SEI message (0=OFF,  1=ON )
+MasterDisplay                   : NULL          # SEI message. Signals SMPTE ST 2086 display info in the format "G(%hu,%hu)B(%hu,%hu)R(%hu,%hu)WP(%hu,%hu)L(%u,%u)" where %hu,%u are unsigned 16bit and 32 bit integers
```

Comment:
we should use 0 instead of NULL.

Reply:
MasteringDisplay is a string parameter. So I added default value as NULL